generator/ServiceClientGeneratorLib/Generators/SourceFiles/StructureGenerator.tt

```diff
@@ -297,11 +297,22 @@ namespace <#=this.Config.Namespace#>.Model
                     }
                     else if (member.IsList && (member.MarshallLocation == MarshallLocation.Header || member.MarshallLocation == MarshallLocation.Headers))
                     {
-                    // see protocol test https://github.com/smithy-lang/smithy/blob/8418149c69be6e6847dffae27e802730e8e40d49/smithy-aws-protocol-tests/model/restXml/http-headers.smithy#L369
-                    // null or empty lists in headers should not be sent over the wire
+                    // restxml and restjson is the only protocol where we send empty lists in headers. See following protocol tests.
+                    // https://github.com/smithy-lang/smithy/blob/main/smithy-aws-protocol-tests/model/restJson1/http-headers.smithy#L427
```

Comment:
Nit: Can we use the permalink as we had before? Line 427 won't always refer to the applicable protocol test.